### PR TITLE
FEAT: added guide for informal reviews

### LINKED ISSUE #245: Guide of informal code reviews on GitHub
Create a Markdown file with GitHub issue template, thoughts, and lessons learned from previous reviews

### PR DESCRIPTION
**Pull request by alaws-USGS**
Closes #245

### CODE REVIEW

File: doc/Notebook_review_guide.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+API calls
+FTP calls
+The following are needed for running this notebook:
+
+Access to the [ESIP-Nebari on AWS](https://nebari.esipfed.org/). If you do not have access to this, please email Andrew at alaws@contractor.usgs.gov and he will get you credentials.
+Github account (though this can be worked around if need be, just let Andrew know)
+Set Up
+
+Navigate to the [hytest-org repo](https://github.com/hytest-org/hytest)
+If you are a member of the `hytest-org` GH organization, create a branch named "initials-NB-review"
```
Comment (gzt5142) [inline doc/Notebook_review_guide.md:83]: Didn't one of the last reviewers try to create a branch with that exact name? "initials-NB-review"...   ??

"create a branch into which your suggestions will go -- such as xx-NN-review, where 'xx' is your initials.

File: doc/Notebook_review_guide.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+Login to ESIP-QHUB and start a server instance.
+Click on the *Git* dropdown from the top navigation bar and select *Clone a repo*. Copy and paste the SSH url from the *hytest-org* repo (or forked repo) into the dialog box and click *Clone*
+Informal Review
+
+Navigate to the hytest/dataset_preprocessing/tutorials/CONUS404 folder and open the CONUS404_DRB_Data_Prep.ipynb, selecting the "users-pangeo" kernel when asked.
+Based on the review type Andrew has asked you for and any additional you wish to do, use the questions as guidelines for any changes and feedback that you make:
+Feedback should be in the format of "[first name] comment/edited text/edited code"
+**Code review**
+Is the code readable? Do variable names make sense? Are all of the packages and tools used? Are functions adequately explained, including docstrings? Is there unnecessary duplication of code? Is there a more effective way to accomplish what the code does? What needs further explanation?
```
Comment (gzt5142) [inline doc/Notebook_review_guide.md:93]: I like that docstrings are mentioned... how do you feel about type annotations?

File: doc/Notebook_review_guide.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+Navigate to the [hytest-org repo](https://github.com/hytest-org/hytest)
+If you are a member of the `hytest-org` GH organization, create a branch named "initials-NB-review"
+If you are not a member, create a fork of the project. Your steps for project contribution on GH are detailed well in [this guide to contributing to projects](https://docs.github.com/en/get-started/quickstart/contributing-to-projects)
+Login to ESIP-QHUB and start a server instance.
+Click on the *Git* dropdown from the top navigation bar and select *Clone a repo*. Copy and paste the SSH url from the *hytest-org* repo (or forked repo) into the dialog box and click *Clone*
+Informal Review
+
+Navigate to the hytest/dataset_preprocessing/tutorials/CONUS404 folder and open the CONUS404_DRB_Data_Prep.ipynb, selecting the "users-pangeo" kernel when asked.
+Based on the review type Andrew has asked you for and any additional you wish to do, use the questions as guidelines for any changes and feedback that you make:
+Feedback should be in the format of "[first name] comment/edited text/edited code"
```
Comment (gzt5142) [inline doc/Notebook_review_guide.md:91]: again -- I'm not sure this is needed. git handles this for you.

File: doc/Notebook_review_guide.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+Please submit all feedback and changes via pull request by January 20th, 2023. Thank you for your help and if you have any questions, please feel free to contact Andrew.
+
+## Notebook Merge Conflicts
+
+When attempting to merge the branches, there is a high likelihood of merge conflicts. This is due to the structure of Jupyter Notebooks, which are an open document format based on JSON. What is seen by the person running the notebook cells are a rendered visualization so changes to a cell may cause issues in the underlying  JSON. Both GitHub and GitLab have write ups for handling merge conflicts. Here is the link to [Github's guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line) and here is [GitLab's guide](https://docs.gitlab.com/ee/user/project/merge_requests/conflicts.html).
```
Comment (gzt5142) [inline doc/Notebook_review_guide.md:107]: I'd invert the first part of this -- 
Emphasize that notebooks are composed largely of text we don't see and can't edit directly.

**Review by gzt5142 — APPROVED** [review]
Here you go.... thanks for explaining your logic earlier today. 

As a rule, I think the standard git mechanism is the right way to do this, but your emphasis on removing those details from reviewers is likely an advantage here.

File: doc/Notebook_review_guide.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+Author: Andrew Laws - USGS Web Informatics and Mapping
+
+Date: 3/15/2023
+
+## Purpose
+
+The purpose of this document is to create a guide for setting up and executing an informal review for Jupyter Notebooks, with an emphasis on Notebooks that serve as tutorials in coding and domain knowledge.
+
+## Informal review process overview
+
```
[the remark this answers is not in the document]
Comment (alaws-USGS) [reply doc/Notebook_review_guide.md:12]: Definite upgrade!

File: doc/Notebook_review_guide.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+What platform will the review be performed on and does the reviewere have access? Who does the reviewer contact to get access? Are there any other accounts the reviewer will need to have?
+
+*Instructions to perform review*
+
+This section will lay out the steps to completing the review. Some important steps are:
+
+1. How to create a branch (if a member of repository organization) or fork the project (if not a member).
+2. How to setup the environment on the platform the review will be done on. If extra instructions are needed to setup a custom environment, be sure to include an environment YML to do so.
+3. How to format feedback in notebook. A common way is "**[first name] comment/edited text/edited code"**.
+
```
[the remark this answers is not in the document]
Comment (alaws-USGS) [reply doc/Notebook_review_guide.md:54]: After doing this, it really helps navigate through the document to find their comments.

**Pull request merged**